examples/hello_worlds.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require 'vedeu'
+
+# An example application to demonstrate 'Hello World' and a minimum of
+# interactivity.  It uses the DSL-style of vedeu (not the full-flegded
+# MVC-approach).
+#
+# If you have cloned this repository from GitHub, you can run this example:
+#
+#     bundle exec ./examples/hello_worlds.rb
+#
+class HelloWorldsApp
+
+  Vedeu.bind(:_initialize_) { Vedeu.trigger(:_refresh_) }
+
+  Vedeu.configure do
```

Comment:
Leave this in for now, but I have fixed this issue and will release a new version shortly.